### PR TITLE
Adding ShadowOS

### PR DESCRIPTION
**Pull request by bugwrangler**
https://www.youtube.com/watch?v=rfshPBeVSoc

### CODE REVIEW

File: Document/0x08-Testing-Tools.md [hunk 1]
```diff
@@ -9,6 +9,7 @@ To perform security testing different tools are available in order to be able to
 * Mobisec - http://sourceforge.net/projects/mobisec/ - Mobile security testing live environment.
 * Santoku - https://santoku-linux.com/ - Santoku is an OS and can be run outside a VM as a standalone operating system.
 * Vezir Project - https://github.com/oguzhantopgul/Vezir-Project - Mobile Application Pentesting and Malware Analysis Environment.
+* ShadowOS - https://saas.hpe.com/en-us/home - Free Virtual OS for testing Android apps security.
```
Comment (sushi2k) [inline Document/0x08-Testing-Tools.md]: Can you please provide the download link? Is this distribution maintained? I couldn't find any documentation or ISO/VM to download.

Comment (Sjord) [inline Document/0x08-Testing-Tools.md]: @vbisbest, do you know more about the current status of ShadowOS?

Comment (vbisbest) [inline Document/0x08-Testing-Tools.md]: ShadowOS has been discontinued by HP.  I am going to look at getting it back up just as an open source project by myself.   I don't have a time frame on that quite yet.

Comment (sushi2k) [inline Document/0x08-Testing-Tools.md]: Thanks for clarifying. @bugwrangler: If you have other information to this topic, please let me know. I will close this PR and not merge it.

**Pull request closed without merging**